src/PlayFabBuddy.Lib/Entities/Accounts/TitlePlayerAccountEntity.cs

```diff
@@ -2,7 +2,7 @@
 
 public class TitlePlayerAccountEntity
 {
-    public string? Id { get; init; }
+    public string? Id { get; set; }
```

Comment:
Why do we need to change this here?

Reply:
Ahhh yes, this change was needed as we need to pass in the Id on the TitlePlayerAccountEntity. This is so that we can ban an individual player.